### PR TITLE
Improve a mutating-state.md's markdown markup.

### PR DESCRIPTION
**Pull request by Ch-Valentine**
Rewrite file from the javascript to a markdown text style

### CODE REVIEW

File: anti-patterns/05.mutating-state.md [hunk 1]
```diff
@@ -1,14 +1,12 @@
...
- */
+# Mutating State without setState()
+- Causes state changes without making component re-render.
+- Whenever setState gets called in future, the mutated state gets applied.
 
-// BAD
+### Reference:
+[React Design Patterns and best practices by Michele Bertoli.](https://github.com/MicheleBertoli/react-design-patterns-and-best-practices)
+
+### BAD
```
Comment (Reda-bx) [inline anti-patterns/05.mutating-state.md]: one more # please

File: anti-patterns/05.mutating-state.md [hunk 2]
```diff
@@ -38,8 +36,10 @@ class SampleComponent extends Component {
     )
   }
 }
+```
 
-// GOOD
+### GOOD
```
Comment (Reda-bx) [inline anti-patterns/05.mutating-state.md]: same here

Comment (Ch-Valentine) [inline anti-patterns/05.mutating-state.md]: @Reda-bx, few seconds)

**Comment by Ch-Valentine** [comment]
@Reda-bx, done.

**Pull request merged**